File: fuelnet_back/src/main/java/com/fuelnet/fuelnet/dto/AlertResponse.java
package com.fuelnet.fuelnet.dto;

import com.fuelnet.fuelnet.enums.FuelType;
import java.math.BigDecimal;
import java.time.LocalDateTime;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class AlertResponse {

    private Long id;
    private FuelType fuelType;
    private BigDecimal levelAtAlert;
    private BigDecimal percentageAtAlert;
    private BigDecimal thresholdUsed;
    private LocalDateTime createdAt;
    private boolean read;
}

File: FuelControl/app/src/main/java/co/edu/unipiloto/fuelcontrol/models/AlertResponse.kt
package co.edu.unipiloto.fuelcontrol.models

import java.math.BigDecimal

data class AlertResponse(
    val id: Long,
    val fuelType: String,
    val levelAtAlert: BigDecimal?,
    val percentageAtAlert: BigDecimal?,
    val thresholdUsed: BigDecimal?,
    val createdAt: String?,
    val read: Boolean
) {
    val isRead: Boolean get() = read
}

